streams/src/main/java/org/apache/kafka/streams/kstream/internals/SessionWindowedKStreamImpl.java

```diff
@@ -97,19 +92,24 @@ public Long apply(final K aggKey, final Long aggOne, final Long aggTwo) {
             materializedInternal.withValueSerde(Serdes.Long());
         }
 
-        return (KTable<Windowed<K>, Long>) aggregateBuilder.build(
-            new KStreamSessionWindowAggregate<>(windows, materializedInternal.storeName(), aggregateBuilder.countInitializer, aggregateBuilder.countAggregator, countMerger),
+        return aggregateBuilder.build(
```

Comment:
Note that we no longer need this unchecked cast because `build` now preserves the knowledge that the aggregate will produce `Windowed<K>` keys.